css/hero.css

```diff
@@ -78,7 +78,7 @@
         text-align: center;
         color: white;
         float: left; 
-        margin-top: calc(50vh - 270px);
+        margin-top: calc(50vh - 110px) !important;
```

Comment:
Išmetam !important, to jei norimas stilius nesuveikia:
- perdėlioti CSS failų eiliškumą, jog komponentai būtų žemiau bendrinio stiliaus;
- arba padaryti didesnio specifiškumo selector'ių.